ojd_daps_skills/utils/bert_vectorizer.py

```diff
@@ -34,14 +32,6 @@ def fit(self, *_):
     def transform(self, texts):
         logger.info(f"Getting embeddings for {len(texts)} texts ...")
         t0 = time.time()
-        if self.multi_process:
-            logger.info(".. with multiprocessing")
-            pool = self.bert_model.start_multi_process_pool()
-            self.embedded_x = self.bert_model.encode_multi_process(
-                texts, pool, batch_size=self.batch_size
-            )
-            self.bert_model.stop_multi_process_pool(pool)
-        else:
-            self.embedded_x = self.bert_model.encode(texts, show_progress_bar=True)
+        self.embedded_x = self.bert_model.encode(texts, batch_size=self.batch_size)
```

Comment:
@Jack-Vines if we allow a multi_process argument in extract skills do you think we could have this as:
```
if self.multi_process:
            logger.info(".. with multiprocessing")
            pool = self.bert_model.start_multi_process_pool()
            self.embedded_x = self.bert_model.encode_multi_process(
                texts, pool, batch_size=self.batch_size
            )
            self.bert_model.stop_multi_process_pool(pool)
        else:
            self.embedded_x = self.bert_model.encode(texts, batch_size=self.batch_size)
```